docs/dev-guide.md

```diff
@@ -115,6 +117,15 @@ Cross-compilation for ARM with Mainflux make:
 GOOS=linux GOARCH=arm GOARM=7 make
 ```
 
+## Running tests
+
+To run all of the tests you can execute:
+```
+make test
+```
+
+Dockertest is used for the test, so to run the tests you will need Docker and NATS running.
```

Comment:
You don't need NATS on your machine to execute tests.

Reply:
thanks for clearance @anovakovic01 
I fixed this and also changed the test target, so that the same `go test` command is executed as in Semaphore CI pull request check